modules/kmm-customizing-upgrades-for-kernel-modules.adoc

```diff
@@ -37,15 +36,30 @@ The update should be atomic; that is, both the `containerImage` and `version` fi
 
```

Comment:
Wondering if the text in this step telling us that the update must be atomic should be an admonition. Will the procedure fail if this step is not done properly? 
You could call this requirement out a bit, something like [NOTE] or [IMPORTANT]:

[IMPORTANT]

> ====

The update must be atomic; that is, both the `containerImage` and `version` fields must be updated 
simultaneously.

> ====

(Ignore the quotes before the "=", this is only way I could keep markdown from making the text ginormous.)

Reply:
No change.